lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -59,11 +60,13 @@ def create_vm(attrs)
             end
             attrs[:memory_policy] = OvirtSDK4::MemoryPolicy.new(:guaranteed => attrs[:memory]) if attrs[:memory].to_i < Fog::Compute::Ovirt::DISK_SIZE_TO_GB
             attrs[:high_availability] = OvirtSDK4::HighAvailability.new(:enabled => attrs[:ha] == "1") if attrs[:ha].present?
+            
+            if attrs[:clone] == true && attrs[:disks].present?
+              process_vm_opts(attrs)
+            end
 
-            # TODO: handle cloning from template
-            process_vm_opts(attrs)
             new_vm = OvirtSDK4::Vm.new(attrs)
-            vms_service.add(new_vm)
+            vms_service.add(new_vm, :clone => attrs[:clone])
```

Comment:
I see that `:clone` is passed as part of attrs. It feels redundant to send it both in `attrs` and as a parameter here.